getting-started/prestashop.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: PrestaShop
+description: Enable PrestaShop to correctly read client IPs from Edgee’s forwarded headers.
```

Comment:
This seems a bit too specific :) 

I understand you want to anticipate the issue, but this is appearing as a sub-title and in my opinion it should be a bit less specific and maybe focus on the potential goals of a Prestashop user.

In other words, it should answer the question: "Can I integrate Edgee with Prestashop?" and the main answer should be "YES ^_^" (instead of "yes, but...").

So I would craft a slightly different subtitle and maybe reshape the first paragraph too.